messaging/src/main/java/org/axonframework/eventhandling/tokenstore/jpa/JpaTokenStore.java

```diff
@@ -257,6 +283,59 @@ protected TokenEntry loadToken(String processorName, int segment, EntityManager
         return token;
     }
 
+    /**
+     * Tries loading an existing token owned by a processor with given {@code processorName} and {@code segment}. If such a token entry exists an attempt will
+     * be made to claim the token. If that succeeds the token will be returned. If the token is already owned by another node an {@link
+     * UnableToClaimTokenException} will be thrown.
+     * <p>
+     * If no such token exists yet, a new token entry will be inserted with {@code null} token owned by this node and return {@code null}.
```

Comment:
I feel this sentence doesn't completely flow. 
I have a suggestion but feel free to take a different course that better fits the description:
```suggestion
     * If no such token exists yet, a new token entry will be inserted with a {@code null} token, owned by this node, and this method returns {@code null}.
```